plugins/woocommerce/client/blocks/assets/js/blocks/featured-items/featured-category/block.json

```diff
@@ -3,15 +3,13 @@
 	"version": "1.0.0",
 	"title": "Featured Category",
 	"category": "woocommerce",
-	"keywords": [
-		"WooCommerce"
-	],
+	"keywords": [ "WooCommerce" ],
 	"description": "Visually highlight a product category and encourage prompt action.",
 	"supports": {
-		"align": [
-			"wide",
-			"full"
-		],
+		"interactivity": {
+			"clientNavigation": false
```

Comment:
Is there a particular reason for this? I checked it and it looks like that the client Navigation works fine.

Reply:
Good question, I actually based that on my previous research where I initially marked [incompatible blocks](https://github.com/woocommerce/woocommerce/pull/45435/files#diff-9858340fd47fd06ff7ee58c76a0a3ffa29bf3e91c3729aaca980516c2c2406c7R7-R11) but need to double check! It was about featured-product but they're connected.

Reply:
You're right. Verified and it should be marked as `clientNavigation: true`. Done in 8b86758ff47e402f69c221795b31320555225e4d